CMakeLists.txt

```diff
@@ -363,6 +363,54 @@ opm_add_test(test_parallelwellinfo_mpi
   NO_COMPILE
     )
 
+opm_add_test(test_parallel_wbp_sourcevalues_np1
```

Comment:
okay did not know we want multiple tests. see how test_parallelwellinfo_mpi vs test_parallelwellinfo is done (ie standard CMakeLists_files.txt entry and then an opm_add_test for the parallel incarnations).

Reply:
> okay did not know we want multiple tests.

Neither did I until I actually started creating the test and realized that multiple tests with different number of ranks/processes might actually be useful...

> see how `test_parallelwellinfo_mpi` vs `test_parallelwellinfo` is done

Thanks for the hint.  I've pushed an update to this PR that mimics that pattern.

Reply:
looks good. still approved.

Reply:
> looks good. still approved.

:+1:

Tangentially, does a "bare" [`opm_add_test()`](https://github.com/OPM/opm-simulators/blob/5cddb59b95ea1d7acb0ffe314e164333a7b6eccc/CMakeLists.txt#L377) call specifying a [runtime number of processes](https://github.com/OPM/opm-simulators/blob/5cddb59b95ea1d7acb0ffe314e164333a7b6eccc/CMakeLists.txt#L383) play well with the CTest `PROCESSORS` property used in PR #4641 to enable more test parallelism?  Do we risk oversubscribing the CI system's computational capacity when we have multiple tests that run MPI-aware executables without telling CTest how many processors they actually use?

It is **probably** not a big problem for this PR per se since each test runs in well under one second, but it maybe it's problem for tests that take more time to run?

Reply:
if you specify PROCESSORS it is [set as a propery on the test](https://github.com/OPM/opm-common/blob/master/cmake/Modules/OpmSatellites.cmake#L361) and all is well.

Reply:
>> does a "bare" [`opm_add_test()`](https://github.com/OPM/opm-simulators/blob/5cddb59b95ea1d7acb0ffe314e164333a7b6eccc/CMakeLists.txt#L377) call specifying a [runtime number of processes](https://github.com/OPM/opm-simulators/blob/5cddb59b95ea1d7acb0ffe314e164333a7b6eccc/CMakeLists.txt#L383) play well with the CTest `PROCESSORS` property used in PR #4641 to enable more test parallelism?
>
> if you specify `PROCESSORS` it is [set as a propery on the test](https://github.com/OPM/opm-common/blob/master/cmake/Modules/OpmSatellites.cmake#L361) and all is well.

Noted.  I've pushed an update to set `PROCESSORS` on the new tests as well.  If the build check goes through I will merge this PR in its current state.

I will make a separate, follow-up PR to set proper `PROCESSORS` values on the other MPI tests which use a bare `opm_add_test()` macro call.